src/Resources/config/core.php

```diff
@@ -0,0 +1,235 @@
+            ->args([
+                new ReferenceConfigurator('sonata.admin.pool'),
+                [],
+                new ReferenceConfigurator('service_container'),
+            ])
+
+        ->alias(AdminPoolLoader::class, 'sonata.admin.route_loader')
+
+        ->set('sonata.admin.helper', AdminHelper::class)
+            ->public()
+            ->args([
+                new ReferenceConfigurator('sonata.admin.pool'),
+            ])
+
+        ->alias(AdminHelper::class, 'sonata.admin.helper')
+
+        ->set('sonata.admin.builder.filter.factory', FilterFactory::class)
+            ->public()
+            ->args([
+                new ReferenceConfigurator('service_container'),
+                [],
+            ])
+
+        ->alias(FilterFactory::class, 'sonata.admin.builder.filter.factory')
+
+        ->alias(FilterFactoryInterface::class, 'sonata.admin.builder.filter.factory')
+
+        ->set('sonata.admin.breadcrumbs_builder', BreadcrumbsBuilder::class)
+            ->public()
+            ->args([
+                '%sonata.admin.configuration.breadcrumbs%',
+            ])
+
+        ->alias(BreadcrumbsBuilder::class, 'sonata.admin.breadcrumbs_builder')
+
+        ->alias(BreadcrumbsBuilderInterface::class, 'sonata.admin.breadcrumbs_builder')
+
+        // Services used to format the label, default is sonata.admin.label.strategy.noop
+
+        ->set('sonata.admin.label.strategy.bc', BCLabelTranslatorStrategy::class)
+            ->public()
+
+        ->alias(BCLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.bc')
+
+        ->set('sonata.admin.label.strategy.native', NativeLabelTranslatorStrategy::class)
+            ->public()
+
+        ->alias(NativeLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.native')
+
+        ->alias(LabelTranslatorStrategyInterface::class, 'sonata.admin.label.strategy.native')
+
+        ->set('sonata.admin.label.strategy.noop', NoopLabelTranslatorStrategy::class)
+            ->public()
+
+        ->alias(NoopLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.noop')
+
+        ->set('sonata.admin.label.strategy.underscore', UnderscoreLabelTranslatorStrategy::class)
+            ->public()
+
+        ->alias(UnderscoreLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.underscore')
+
+        ->set('sonata.admin.label.strategy.form_component', FormLabelTranslatorStrategy::class)
+            ->public()
+
+        ->alias(FormLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.form_component')
+
+        // NEXT_MAJOR: Remove this service.
+        ->set('sonata.admin.translator.extractor.jms_translator_bundle', DeprecatedAdminExtractor::class)
+            ->public()
+            ->tag('jms_translation.extractor', [
+                'alias' => 'sonata_admin',
+            ])
+            ->deprecate(sprintf(
+                'The service "%%service_id%%" is deprecated in favor of the "%s" service since version 3.72 and will be removed in 4.0.',
```

Comment:
```suggestion
                'The service "%%service_id%%" is deprecated since sonata-project/admin-bundle 3.72 and will be removed in 4.0. Use "%s" service instead.',
```